prince-of-versions/src/main/java/co/infinum/princeofversions/PrinceOfVersionsConfig.java

```diff
@@ -122,7 +123,7 @@ public String toString() {
         private NotificationType optionalNotificationType;
 
         @Nullable
-        private Map<String, Object> metadata;
+        private Map<String, Object> metadata = Collections.emptyMap();
```

Comment:
Have you tried it out?
Read this: https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#emptyMap()
According to documentation emptyMap gives you immutable instance that throws exception in put methods.

Use new HashMap instead.

Reply:
Also, I've noticed that tests aren't running on bitrise. Please add the tests to run also so we would notice problems.